src/preset_cli/api/clients/preset.py

```diff
@@ -61,13 +63,24 @@ def get_team_members(self, team_name: str) -> List[Any]:
         """
         Retrieve all users for a given team.
         """
-        url = self.get_base_url() / "teams" / team_name / "memberships"
-        _logger.debug("GET %s", url)
-        response = self.session.get(url)
-        validate_response(response)
+        users = []
+        page_number = 1
+        while True:
+            params = {"page_number": page_number, "page_size": MANAGER_MAX_PAGE_SIZE}
+            url = (
+                self.get_base_url() / "teams" / team_name / "memberships" / "" % params
```

Comment:
Oh, do we need the trailing slash here, before `params`?

Reply:
We don't, actually. Both would work -- lemme remove them.